src/syntax/ppx_sedlex.ml

```diff
@@ -284,13 +284,15 @@ let regexp_of_pattern env =
         List.fold_left (fun r p -> Sedlex.seq r (aux p))
           (aux p)
           pl
-    | Ppat_construct ({txt = Lident "Star"}, Some p) ->
+    | Ppat_construct (_, Some (_ :: _, _)) ->
+        err p.ppat_loc "Explicit binders for type variables not supported"
```

Comment:
Maybe same comment as in https://github.com/ocaml-ppx/ppx_deriving_yojson/pull/142#issuecomment-1080543029

Reply:
Hey @kit-ty-kate, thanks for the review! See my answer [here](https://github.com/ocaml-ppx/ppx_deriving_yojson/pull/142#issuecomment-1080635196). If you're sure that the good new behavior here is to ignore the new feature, let me know and I update the PR! (I'm asking because of your "maybe")